webapp/tests/elster_client/mock_erica.py

```diff
@@ -18,7 +18,7 @@
                                             "person_a_first_name", "person_a_religion", "person_a_street",
                                             "person_a_street_number", "person_a_plz", "person_a_town", "person_a_blind",
                                             "account_holder"]
-_METADATA_KEYS = ["year", "is_digitally_signed"]
+_METADATA_KEYS = ["year"]
```

Comment:
I am wondering if we should raise an error for incorrect year data in the mock erica. Because we only use the MockErica if we run our web tests? I think we would also get some kind of error from Erica for incorrect years or not?

Reply:
Uh, that is a good point. No, we do not check that yet. We should though. I'll add it. A check in erica_input, right?

Reply:
That's not what I meant but also a great idea. I think we should definitely add a check in mock_erica

Reply:
Did it^^